rules/macos/persistence_emond_rules_file_creation.toml

```diff
@@ -59,8 +59,8 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-file where host.os.type == "macos" and event.type != "deletion" and
- file.path : ("/private/etc/emond.d/rules/*.plist", "/etc/emon.d/rules/*.plist", "/private/var/db/emondClients/*")
+file where host.os.type == "macos" and event.action == "modification" and
+ file.path like ("/private/etc/emond.d/rules/*.plist", "/etc/emon.d/rules/*.plist", "/private/var/db/emondClients/*")
```

Comment:
Does this need to be `like~` since we use wildcards?

Reply:
Yeah it works with wildcards. Its just case sensitive.